app/controllers/api/scores_controller.rb

```diff
@@ -5,7 +5,19 @@ class ScoresController < ApplicationController
     before_action :validate_score_user_id, only: :destroy
 
     def user_feed
-      scores = Score.all.order(played_at: :desc, id: :desc)
+      scores = Score.all.limit(25).order(played_at: :desc, id: :desc).includes(:user)
```

Comment:
1. Remove the score limit from this PR as this is a requirement for another PR.
2. Also, it's not the role of this PR to fix the n+1 query issue. It will bring git conflicts when merging this PR with the first one.